src/chrome/content/rules/SourceForge.net.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+	Related:
+		SF.net.xml
+		SourceForge.jp.xml
+
+	Failed:
+		(projectname).sourceforge.net
+		e.g. http://doublecmd.sourceforge.net/
+
+	Mismatched:
+		*.cvs.sourceforge.net
+		*.git.sourceforge.net
+		heanet.dl.sourceforge.net
+		p.sourceforge.net
+
+	Refused:
+		*.hg.sourceforge.net
+-->
+
+<ruleset name="SourceForge.net">
+	<!--	Directly:	-->
+	<target host="sourceforge.net" />
+	<target host="www.sourceforge.net" />
+	<target host="apps.sourceforge.net" />
+	<target host="deals.sourceforge.net" />
+	<target host="netcologne.dl.sourceforge.net" />
+	<target host="jaist.dl.sourceforge.net" />
+	<target host="netix.dl.sourceforge.net" />
```

Comment:
A few other examples:
excellmedia.dl.sourceforge.net
svwh.dl.sourceforge.net
superb-sea2.dl.sourceforge.net
ufpr.dl.sourceforge.net

I think this will work for any download mirror.